orbs/pulumi.yml

```diff
@@ -90,6 +90,34 @@ commands:
           name: "pulumi stack rm << parameters.stack >>"
           command: pulumi stack rm << parameters.stack >> <<# parameters.force >>--force<</ parameters.force >> --cwd << parameters.working_directory >>
 
+  # stack output command
+  stack_output:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to operate on."
+      property_name:
+        type: string
+        description: "Property name of the stack output to set to the environment variable."
+      env_var:
+        type: string
+        description: "Name of the environment variable to set the output property's value to."
+      show_secrets:
+        type: boolean
+        description: "Display stack outputs which are marked as secret in plaintext."
+        default: false
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack output << parameters.property_name >> --stack << parameters.stack >>"
+          command: | 
+            OUTPUT_VALUE=$(pulumi stack output << parameters.property_name >> --stack << parameters.stack >> <<# parameters.show_secrets >>--show-secrets<</ parameters.show_secrets >> --cwd << parameters.working_directory >>)
+            echo "export << parameters.env_var >>=\"${OUTPUT_VALUE}\"" >> $BASH_ENV
```

Comment:
In order to interpolate the environment variable, the content of `echo` needs to be double quoted (not single quoted). This has been fixed.